packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx

```diff
@@ -266,8 +270,12 @@ export class ChromeService {
 
     const getIsNavDrawerLocked$ = isNavDrawerLocked$.pipe(takeUntil(this.stop$));
 
+    const activeSolutionNavId$ = projectNavigation
```

Comment:
Can we keep this inside the ProjectNavigation class and return it from there (`getActiveSolutionNavId$`)?